model/src/form/form-metadata/index.ts

```diff
@@ -28,11 +33,31 @@ export const teamEmailSchema = Joi.string()
  * Joi schema for `FormMetadata` interface
  * @see {@link FormMetadata}
```

Comment:
Thanks @davidjamesstone!

Mind doing a quick review of these `@link` tags?

E.g. This one is my fault, it should have been `FormMetadataInput`

```suggestion
 * @see {@link FormMetadataInput}
```

Reply:
Fixed with 3bbb93aa4f770999ea2292070f7183da5e99b323